token/cli/src/main.rs

```diff
@@ -213,12 +213,20 @@ fn command_burn(config: &Config, source: Pubkey, ui_amount: f64) -> CommmandResu
         .rpc_client
         .get_token_account_balance_with_commitment(&source, config.commitment_config)?
         .value;
+    let source_account = config
```

Comment:
`source_token_balance` is only used to figure decimals, which we can get from the mint.  So I think we can remove the `get_token_account_balance_with_commitment()` RPC call entirely

Reply:
Still two calls, since we'll have to pull the mint.

Reply:
ah, yeah ok.  wfm